panther_gazebo/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(panther_gazebo)
+
+find_package(catkin REQUIRED)
+
+catkin_package()
+
+install(DIRECTORY 
+	launch
+	DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+)
```

```suggestion
install(DIRECTORY 
  launch
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
)
```
match other packages formatting